### PR TITLE
Add Indonesia (id) locale

### PR DESCRIPTION
**Pull request by dwisiswant0**
Fixes #

## Proposed Changes

* Add Indonesia (id) language

**Release Note**

```release-note
 - Indonesia (id) language translation
```

### CODE REVIEW

**Comment by dwisiswant0** [comment]
@googlebot I signed it!

**Comment by mikehelmick** [comment]
Thanks for the contribution!

/ok-to-test
/hold

Checking on engagement with Indonesia.

**Review by sethvargo — DISMISSED** [review]

File: internal/i18n/locales/id/default.po [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#
+# navigation menus
+# ----------------
+
+msgid "nav.issue-code"
+msgstr "Kode masalah"
```
Comment (sethvargo) [inline internal/i18n/locales/id/default.po:14]: Hey @dwisiswant0 thank you for the PR. I think this translation might be incorrect. "Issue" is a verb in this case (synonyms: create, generate, make). I think this translation to "trouble".

Comment (sethvargo) [reply internal/i18n/locales/id/default.po:14]: (For all instances of "issue"). I'm not a native speaker though, so please correct me if I'm wrong

Comment (dwisiswant0) [reply internal/i18n/locales/id/default.po]: Thanks for review this, looks like your context is correct. Should be resolved as per 63b4fee.

**Comment by sethvargo** [comment]
Thank you!

**Pull request merged**